images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -133,7 +133,10 @@ function Get-VcpkgVersion {
     $result = Get-CommandResult "vcpkg version"
     $result.Output -match "version (?<version>\d+\.\d+\.\d+)" | Out-Null
     $vcpkgVersion = $Matches.version
-    return "Vcpkg $vcpkgVersion"
+    Push-Location -Path "/usr/local/share/vcpkg"
+    $commitId = git rev-parse master
```

Comment:
nitpick:
1) it is better to use `git rev-parse HEAD` instead of `master`. `HEAD` points to the current branch and it is more correct way than `master` branch.
2) I think short commit should be enough here: `git rev-parse --short HEAD`

Reply:
thank you, changed to short one